cmd/test/main_test.go

```diff
@@ -240,6 +252,49 @@ func test_testCommand_version_flags_validation(t *testing.T, ctx *TestCommandCon
 	assert.NoError(t, err)
 }
 
+func test_testCommand_empty_dir(t *testing.T, readerMock *ReaderMock, ctx *TestCommandContext) {
+	emptyDir := t.TempDir()
+	emptyDirPaths := filepath.Join(emptyDir, "*.yaml")
+
+	readerMock.On("FilterFiles", []string{emptyDirPaths}).Return([]string{}, nil)
+	err := executeTestCommand(ctx, []string{emptyDirPaths})
+
+	assert.EqualError(t, err, "No files detected")
+	readerMock.AssertCalled(t, "FilterFiles", []string{emptyDirPaths})
+}
+
+func test_testCommand_no_valid_k8s_resources(t *testing.T, readerMock *ReaderMock, evaluator *mockEvaluator, k8sValidator *K8sValidatorMock, filesConfigurations []*extractor.FileConfigurations, printerMock *PrinterMock, ctx *TestCommandContext) {
```

Comment:
I'm not sure that I understand this test case 🥺. I think should test that all files fail on K8s schema validation. WDYT?

Reply:
oops. I also had doubt that the function name is not descriptive (i.e. the function name does not tell what test case its addressing). Could you kindly suggest another name for this test case? 

this test function addresses the test-case for "Given path contains YAML files that are not Kubernetes resources"

Reply:
Agree :) I also suggest specifying what we expect the functions to be called with.